src/Constants/global.js

```diff
@@ -0,0 +1,10 @@
+const tracklist = [
```

Comment:
Is this just for testing?

Not a big deal for a personal project. If this was an enterprise application I'd want stuff like this cleaned up or moved into a testing resources directory

Reply:
Relic from before the Spotify API calls worked. Deleted.